src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java

```diff
@@ -36,7 +36,7 @@
     //
     public static final String SAVED_FLAGS_FILENAME = "savedflags.dat";
     // See https://github.com/TotalFreedom/License - None of the listed names may be removed.
-    public static final List<String> DEVELOPERS = Arrays.asList("Madgeek1450", "Prozza", "Wild1145", "WickedGamingUK", "aggelosQQ");
+    public static final List<String> DEVELOPERS = Arrays.asList("Madgeek1450", "Prozza", "Wild1145", "WickedGamingUK", "aggelosQQ", "marcocorriero", "Commodore64x");
```

Comment:
Incorrect. As I've stated countless times on other discussions, this is a list of TFM developers by project, not TF Developers.